### PR TITLE
Removed unnecessary codes and modify a code lines to support to UNIT coin.

### PR DESCRIPTION
**Pull request by jeong760**
Removed unused Preinfo.cs class code from on /src/CoiniumServ/Daemon/Responses

Modify or add a code to support a UNIT(Universal Coin)

Add a new PropertyConvertor.cs class on /srv/CoiniumServ/Daemon/Converters/PropertyConverter.cs

Add a following code on MininInfo.cs on /srv/CoiniumServ/Daemon/Responses

[JsonProperty("netmh/s")] // added to support unit and possibly some other coins
private double NetMHps { get; set; }
Modify a following code on src\CoiniumServ\Daemon\DaemonBase.cs

using CoiniumServ.Daemon.Converters;
// process response with converter. needed for all coin wallets which gives non-standard info.
string jsonLC = PropertyConverter.DeserializeWithLowerCasePropertyNames(json).ToString();
_logger.Verbose("rx: {0}", jsonLC.PrettifyJson());
try
{
return JsonConvert.DeserializeObject<DaemonResponse>(jsonLC);
}

Specially thanks to @flyx2888 to modify and add a new class for UNIT coin.

Regards,
John Ahn

### CODE REVIEW

**Comment by jeong760** [comment]
@bonesoul  I've opened a new PR for previous change. please apply this PR

**Comment by bonesoul** [comment]
okay lot better.

thanks

**Comment by bonesoul** [comment]
though the tests are failing;

https://circleci.com/gh/bonesoul/CoiniumServ/281?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

`CSC : error CS2001: Source file '/temp/src/CoiniumServ/Daemon/Converters/PropertyConverter.cs' could not be found. [/temp/src/CoiniumServ/CoiniumServ.csproj]`

i guess u forgot to include the file?

**Comment by jeong760** [comment]
@bonesoul  Sorry. it was remaining on untracked file status.  I forgot to issue a **git add .**

**Comment by jeong760** [comment]
@bonesoul 

Just my private opinion but I want to call this of CoiniumServ as ver 2.6.4 to fixed various issues.
Because after you released a version 2.6.3 we've fixed various issues on it.

File: src/web/default/views/donate/donate.cshtml [hunk 1]
```diff
@@ -30,7 +30,7 @@
                 You can directly donate to our following addresses;<br /><br />
                 <ul>
                     <li>Bitcoin: <a href="bitcoin:1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1?label=CoiniumServ%20Donation"><code>1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1</code></a></li>
-                    <li>Litecoin: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
+                    <li>Ethereum: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
```
Comment (flyx2888) [inline src/web/default/views/donate/donate.cshtml]: @jeong760 

This is still wrong. The eth address in the href does not match the displayed address. It's set to the litecoin address.

Comment (jeong760) [reply src/web/default/views/donate/donate.cshtml]: @flyx2888 Yes but donation page displayed an Ethereum address. so this, not the issue.
capture

Comment (flyx2888) [reply src/web/default/views/donate/donate.cshtml]: @jeong760 

It's not an issue if they manually copy and paste. It's an issue if they click on the link.

**Comment by jeong760** [comment]
@bonesoul,

Do you any reasons to hesitate to merged this PR?

Regards,
John Ahn

**Comment by bonesoul** [comment]
@jeong760 nope i totally forgot about it, lemme check it and merge then if it's all okay.

**Pull request merged**